### DISCUSSION TITLE
where to report player?

### DISCUSSION THREAD

**Discussion by ghost** [chromiecraft/chromiecraft#2349] [General]
Where to report a player for something that happened ingame?

**Comment by 55Honey** [comment]
Hi theo1143,

If a player has broken a rule (see https://www.chromiecraft.com/rules/) You can report them in a support ticket on our [discord](https://discord.gg/chromiecraft).
Since it is asked a lot: We do not accept reports for any kind of item or roll disagrements. (Often referred to as ninja)

**Reply by ghost** [reply]
Ah, damn, I really don't like discord to say the least.
Any otherway?

Yea this is about something ingame, someone that joined a random dungeon.

I was tanking the dungeon, he was the healer.
He was lvl 25 and it was deadmines so to low for hes liking.
So he started to beg for us to kick him out so he could basicly cheat deserter and reque again for another dungeon.
I slowly started to pull and he healed so I thought he was done about it.
Then half way in he just kept going on about that we HAVE TO kick him, so he started to dps to.
Draining hes mana on nothing so we wiped and then he started to rage against me, the tank, for not giving him a mana break.
It just kept getting worse, after that he started with trying to insult me so I was quickly done with that.
In the end he kicks me, and the party seeing me trying to kick him the whole dungeon goes ahead with it.
Then he started to rage/insult me over whisper and before I could even repply quickly ignore me.

All this only to get kicked from a dungeon that you, freely join, a random dungeon that is.
But yea not everyone sees it that way, I guess... (leaving names out of it, cus of public post)

**Reply by 55Honey** [reply]
If you don't like the way a player acts, please use ignore to prevent future RDF matches.
GMs and Staff do not engage in Arguments how to play the game (right).

**Reply by ghost** [reply]
So abusing the vote to kick system to cheat deserter is allowed.
And then harrasment is also allowed, ok glad to know.

P.S. Abusing the ignore system, eke making new accounts to keep harrasing someone is also allowed?
This is a joke, shame it ain't funny.

**Reply by 55Honey** [reply]
I have neither stated that abuse nor that harassment is allowed. 

If you want to report a player for breaking our [rules ](https://www.chromiecraft.com/rules/)(which clearly state what's forbidden in chat), you can do that in a ticket on our discord server. Please provide evidence, e.g. as screenshots, so we are able to evaluate the case and take action.

**Reply by ghost** [reply]
You did, "If you don't like the way a player acts, please use ignore" and rdf doesn't work with ignore, it just teams u up.
Beside that being harrashed is normal and accepted by you guys but once I call someone out as a respond i'm getting a warning through mail?!?
hahaha this server is such a joke, go bann me, like i give a sht about a server that is unable to communicate with their people.
Not monitoring anything at all, and if they do they just go around banning and muting random people for no reason what so ever.
You people are a shame

**Reply by 55Honey** [reply]
There are some wrong assumptions in that reply.
1) RDF does not team you with an ignored player. Using ignore prevents you from being paired again.
2) If you receive a warning in a mail for your chat then that's on your own improper communication. 
3) If you've chosen to ignore our offer to report players who break our rules and instead go forth and insult them, there's no point in acting surprised about retaliation.

I see you don't like our approach to act on reports and would like to harrass people for not playing the way you wish instead. Please do that somewhere else. Or even better, not at all.

Farewell.

**Reply by ghost** [reply]
1, Actually got grouped with people I got on me ignore list several times...
2, I don't rage or insult people unless they started with it, so your not reading logs but relly on screenshots.
3, Ignore? Ignore that there is a ingame ticket system and so hard is it not to make a forum and have it on there.

Ye sure now i'm the one who doesn't like the way others play, that is why they started to rage at me in the first place?
Its a real honey turn around, as always

**Comment by 55Honey** [comment]
After several instances of severe insults, harrassment and racial slurrs, ignored warnings and finally venting with any and all bad words they know at staff/GMs in a ticket, the player was banned permanently on all platforms.
